apps/web/src/api/hooks/notification-templates/useCreateDigestDemoWorkflow.ts

```diff
@@ -0,0 +1,66 @@
+import { useCallback } from 'react';
+import { useMutation } from '@tanstack/react-query';
+import { useNavigate } from 'react-router-dom';
+import { ICreateNotificationTemplateDto, INotificationTemplate, StepTypeEnum } from '@novu/shared';
+
+import { createTemplate } from '../../notification-templates';
+import { parseUrl } from '../../../utils/routeUtils';
+import { ROUTES } from '../../../constants/routes.enum';
+import { errorMessage } from '../../../utils/notifications';
+import { useNotificationGroup } from '../../../hooks';
+
+export const useCreateDigestDemoWorkflow = () => {
```

Comment:
moved the code to reusable hook, used on the getting started page and on the templates placeholder